packages/headless/src/features/sort-criteria/criteria.ts

```diff
@@ -26,6 +62,10 @@ export type SortCriterion =
   | SortByField
   | SortByNoSort;
 
+/**
+ * Allows to build a sort expression that can be understood and executed by the Coveo platform.
+ * @param {SortCriterion} criterion The criterion to translate to a valid sort query expression.
```

Comment:
criterion is repeated

Reply:
The first one is for "the param name".

So, it reads as:
a param named `criterion` of the type `{SortCriterion}` : The criterion to translate...